src/AwsS3Adapter.php

```diff
@@ -456,19 +456,23 @@ public function readStream($path)
         return $response;
     }
 
-    /**
-     * Read an object and normalize the response.
-     *
-     * @param string $path
-     *
-     * @return array|bool
-     */
-    protected function readObject($path)
+  /**
+   * Read an object and normalize the response.
+   *
+   * @param string $path
+   * @param bool   $stream
+   * @return array|bool
+   */
+    protected function readObject($path, $stream)
```

Comment:
Can you set the default value to `true` here? Otherwise, this is a BC break, since it's protected.

Reply:
You are totally right. Fixed.